test/plugin/base_test.rb

```diff
@@ -607,32 +607,43 @@ def test_sanitize_tags_from_container_name_with_require_valid_tags_false
 
   def test_timestamps
     setup_gce_metadata_stubs
-    [Time.at(123_456.789), Time.at(0), Time.now].each do |ts|
+    current_time = Time.now
+    current_year = current_time.year
+    future_before_end_of_year = Time.utc(current_year, 12, 31, 23, 59, 59)
```

Comment:
Same here -- `DateTime` is likely your friend...

Reply:
Changed.